### DISCUSSION TITLE
View port in grafana dashboard

### DISCUSSION THREAD

**Discussion by GollaSaiVenkatesh** [grafana/grafana#70880] [Dashboards, Panels & Visualizations]
Is it possible to set the viewport in the Grafana dashboard? I want the dashboard to auto-adjust column width depending on the viewing device.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82591. Feel free to continue the discussion around this there. Thank you!
